.github/workflows/generate-query-help-docs.yml

```diff
@@ -1,10 +1,17 @@
 name: Generate CodeQL query help documentation using Sphinx
 
 on:
+  workflow_dispatch:
+      inputs:
+        description:
+          description: A description of the purpose of this job. For human consumption.
+          required: false
+        ref: 
+          description: The branch in github/codeql to checkout when generating query help.
+          required: false
+          default: 'lgtm.com'
```

Comment:
You don't need this: `ref` is already a part of the `workflow_dispatch` event. To run against the `lgtm.com` branch, just choose that branch in the dispatch UI.

```suggestion
```